package.json

```diff
@@ -19,11 +19,11 @@
     }
   },
   "scripts": {
-    "test": "npm run lint && node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));' src/index.test.ts",
+    "test": "node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));' src/index.test.ts",
     "build:esm": "tsc --project tsconfig.json --declaration && mv ./dist/esm/index.d.ts ./dist/esm/index.d.mts",
     "build:umd": "tsc --project tsconfig.umd.json --declaration && mv ./dist/umd/index.js ./dist/umd/index.cjs && mv ./dist/umd/index.d.ts ./dist/umd/index.d.cts",
     "build": "npm run build:esm && npm run build:umd",
-    "lint": "eslint src --ext .ts",
+    "lint": "eslint .",
```

Comment:
Please restore this to say `src/`

Reply:
But don't we want to lint all files, regardless of directory? (E.g., if we add `esbuild.js` or `Gruntfile.js`, or if we add a directory called `test`). `dist` and `node_modules` are already excluded.

Reply:
I suppose that would be true if we did that, but why would we do that?

Reply:
I mean, I can merge as-is 🤷

Reply:
> why would we do that?

I assume you mean add new files? Not sure, but for the multiformats package I needed a build script. For this package we can get away with `mv`, but we might also need a build script to handle the dual TypeScript build. I can revert this change if you feel strongly that it must be `src`, but I see no need make it so specific.